src/TLuaInterpreter.cpp

```diff
@@ -10739,6 +10750,501 @@ int TLuaInterpreter::restartIrc(lua_State* L)
     return 1;
 }
 
+#ifdef QT_TEXTTOSPEECH_LIB
+
+/** ttsSpeak( text )
+ *  Synthesizes text for reading.
+ */
+int TLuaInterpreter::ttsSpeak(lua_State* L)
```

Comment:
I like this function - allows someone to just get the text said without all of the complexity of managing queues.

Reply:
Mhm, the only problem with this function is it interrupts the current speech line, which is useful in certain situations but not what you want to do all the time. ttsQueueSpeech() speaks automatically as well, it just takes into account if something's already being said.